addons/http_routing/models/ir_http.py

```diff
@@ -427,8 +427,12 @@ def _dispatch(cls):
             request.is_frontend = func.routing.get('website', False)
         except werkzeug.exceptions.NotFound as e:
             # either we have a language prefixed route, either a real 404
-            # in all cases, website processes them
-            request.is_frontend = True
+            # in all cases, website processes them exept if second element is static
+            # Checking static will avoid to generate an expensive 404 web page since
+            # most of the time the browser is loading and inexisting assets or image. A standard 404 is enough.
+            # Earlier check would be difficult since we don't want to break data modules
+            split = request.httprequest.path.split('/')
+            request.is_frontend = len(split) < 2 or split[1] != 'static'
```

Comment:
If this is for `/module_name/static/whetever`, `split[1]` will be `module_name` and not `static` no?

Reply:
Obviously I made some quick assumption that it was working when it wasn't. New version pushed, thanks for the help.

Reply:
Always happy to help, especially if it is website related ;-)